src/main/java/com/princexml/wrapper/Prince.java

```diff
@@ -316,34 +430,64 @@ private boolean readMessagesFromStderr(Process process) throws IOException {
     }
 
     //region Input options.
+    /**
+     * Rather than retrieving documents beginning with {@code URL}, get them
+     * from the local directory {@code DIR}. This method can be called more than
+     * once to add multiple remappings.
+     * @param remap The remapping "URL=DIR".
+     */
     public void addRemap(String remap) {
         this.remaps.add(remap);
     }
 
+    /**
+     * Clear all of the remappings accumulated by calling {@link #addRemap(String)}.
+     */
     public void clearRemaps() {
         this.remaps.clear();
     }
 
+    /**
+     * Disable access to local files. Enabled by default unless explicitly disabled.
+     * @param noLocalFiles true to disable local files.
+     */
     public void setNoLocalFiles(boolean noLocalFiles) {
         this.noLocalFiles = noLocalFiles;
     }
     //endregion
 
     //region CSS options.
+    /**
+     * Specify the page size.
+     * @param pageSize The page size to use (e.g. "A4").
+     */
     public void setPageSize(String pageSize) {
         this.pageSize = pageSize;
     }
 
+    /**
+     * Specify the page margin.
+     * @param pageMargin The page margin to use (e.g. "20mm").
+     */
     public void setPageMargin(String pageMargin) {
         this.pageMargin = pageMargin;
     }
     //endregion
 
     //region PDF output options.
+    /**
+     * Disable system fonts in the PDF output. Only fonts defined with {@code font-face}
```

Comment:
Disabling system fonts will also affect raster output, so perhaps just "disable the use of system fonts".